imap_processing/codice/codice_l1a.py

```diff
@@ -296,21 +275,22 @@ def unpack_science_data(self, packets: list):
 
         Parameters
         ----------
-        packet : list[space_packet_parser.parser.Packet]
-            List of packets for the APID of interest
+        science_values : str
+            A string of binary data representing the science values of the data
         """
         self.compression_algorithm = LO_COMPRESSION_ID_LOOKUP[self.view_id]
         self.collapse_table_id = LO_COLLAPSE_TABLE_ID_LOOKUP[self.view_id]
 
-        science_values = packets[0].data["DATA"].raw_value
-
+        # TODO: Turn this back on after SIT-3
+        # For SIT-3, just create appropriate length data arrays of all ones
         # Divide up the data by the number of priorities or species
-        num_bits = len(science_values)
-        chunk_size = len(science_values) // self.num_counters
-
-        self.data = [
-            science_values[i : i + chunk_size] for i in range(0, num_bits, chunk_size)
-        ]
+        # science_values = packets[0].data["DATA"].raw_value
+        # num_bits = len(science_values)
+        # chunk_size = len(science_values) // self.num_counters
+        # self.data = [
+        #     science_values[i : i + chunk_size] for i in range(0, num_bits, chunk_size)
+        # ]
+        self.data = ["1" * 128 for _ in range(self.num_counters)]
```

Comment:
Are you making this all ones, 128 * num_counters length? Maybe: `["1"] * 128 * self.num_counters` here.